diff --git a/es/index.js b/es/index.js
index 51c4220a4c89d771ad8c2064fd4f7f18cf04527f..6e9f669feb0116c60bf69aa98f3479573a53ddd7 100644
--- a/es/index.js
+++ b/es/index.js
@@ -4,7 +4,7 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
     else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
     return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
-import { Basecoat, ModifierKey, CssLoader, } from '@antv/x6';
+import { Basecoat, CssLoader, } from '@antv/x6';
 import { SelectionImpl } from './selection';
 import { content } from './style/raw';
 import './api';
